vendor/overwrite-qunit-dom-root-element.js

```diff
@@ -1,6 +1,6 @@
 Object.defineProperty(QUnit.assert.dom, 'rootElement', {
   get: function() {
-    return document.querySelector('#ember-testing');
+    return document.querySelector('#ember-testing > .ember-view');
```

Comment:
Can we make this fallback to the old behavior if the new one can't be found?

Reply:
You mean something like `return document.querySelector('#ember-testing > .ember-view') || document.querySelector('#ember-testing')`?

Reply:
yep, exactly

Reply:
Done! Also reverted the changes in the other test, as suggested!